test/bigtest/tests/funds/FundTags/FundTags-test.js

```diff
@@ -0,0 +1,78 @@
+import { beforeEach, describe, it } from '@bigtest/mocha';
+import { expect } from 'chai';
+
+import setupApplication from '../../../helpers/setup-application';
+import FundDetailsInteractor from '../../../interactors/funds/FundDetails';
+import TagsPaneInteractor from '../../../interactors/common/tags/TagsPane';
+
+const tags = ['tag1', 'tag2'];
+
+describe('Fund details tags', () => {
+  setupApplication();
+
+  const fundDetails = new FundDetailsInteractor();
+  const tagsPane = new TagsPaneInteractor();
+
+  beforeEach(async function () {
+    const fund = this.server.create('fund');
+
+    fund.fund.tags = { tagList: tags };
```

Comment:
IMHO, it's better to handle in common components also, at least to specify there an empty array.

Reply:
Updated fund factory in `stripes-acq-components`